src/onINP.ts

```diff
@@ -63,21 +63,14 @@ export const INPThresholds: MetricRatingThresholds = [200, 500];
  */
 export const onINP = (
   onReport: (metric: INPMetric) => void,
-  opts?: ReportOpts,
+  opts: ReportOpts = {},
 ) => {
   // Return if the browser doesn't support all APIs needed to measure INP.
-  if (
-    !(
-      'PerformanceEventTiming' in self &&
-      'interactionId' in PerformanceEventTiming.prototype
-    )
-  ) {
+  const eventTimingProto = globalThis.PerformanceEventTiming?.prototype;
+  if (!(eventTimingProto && 'interactionId' in eventTimingProto)) {
```

Comment:
this is a funny one

Reply:
`Object.hasOwn()` would probably be worse because you'd still need a fallback in the `undefined` case. :shipit:

Reply:
Yeah, I tried a few iterations of this and I couldn't come up with anyone I liked. This one at least minified the best :)